pages/glossary/content-that-becomes-visible.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Content that becomes visible
+key: content-that-becomes-visible
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+The _content that becomes visible_ is the [root][] element, if one exists, of the [tree][] that contains all elements that, as result of an [event][] [firing][], meet any of the following:
```

Comment:
I'm not too sure what is going on here. How can a tree not have a root element? Do you maybe mean to find the closest common ancestor? How do I know that something is the "result of an event firing"? I don't think we should try to figure out if something is or isn't the result of an event. I think it's sufficient to determine that the content appears within half a second of firing the event, and that the content part of a region that updates without user input.

Reply:
The "if one exists" was meant to apply to the tree, not the root element. My bad.

And I'll update to use the half a second instead of resulting from. That's definitely more testable!